README.md

```diff
@@ -46,7 +46,10 @@ xXl2Nm/u3cPP/eQVrZz5H8eACwIv+LL1EV+9uLanWUa+IO5hHr3KElvKNKD6vN0=
 
 ### GNU Compiler
 ```
+    # Ubuntu / Debian
     sudo apt-get install libmicrohttpd-dev libssl-dev cmake build-essential libhwloc-dev
+    # Fedora
```

Comment:
If you are adding fedora here you need to change line 45 and remove the tezt in the breakets